spartan/aztec-network/eth-devnet/config/mnemonics.yaml

```diff
@@ -0,0 +1,3 @@
+# Used with the eth2-testnet-generator
```

Comment:
I dont think we need to worry about this being public, it just controls our validators